ledger.go

```diff
@@ -19,33 +19,47 @@ const (
 	INSPromptPublicKey    = 0x02
 	INSPromptExtPublicKey = 0x03
 	INSSignHash           = 0x04
+	INSSignTransaction    = 0x05
+	MaxApduSize           = 230
 )
 
+func (l *Ledger) SendToLedger(cla byte, ins byte, p1 byte, p2 byte, buffer []byte) ([]byte, error) {
```

Comment:
The length of the buffer should never overflow a byte since I have capped it at 230 (MaxApduSize) in PrepareChunks. The length also needs to be included in the buffer as it is checked in the Exchange function provided by Zondax.